extensions/binding/org.eclipse.smarthome.binding.ntp/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ntp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ntp">
+		<label>Date</label>
+		<description>NTP Updated Date &amp; Time</description>

```

Comment:
why client and not server? The Thing that provides the info is the server itself, you only implemented a client within your binding to get the time from it.